src/libs/ajax/User.ts

```diff
@@ -217,12 +217,16 @@ export const User = (signal?: AbortSignal) => {
     },
 
     getEnterpriseFeatures: async (): Promise<string[]> => {
-      const res = await fetchSam(
-        '/api/resources/v2?format=flat&resourceTypes=enterprise-feature&roles=user',
-        _.mergeAll([authOpts(), { signal }])
-      );
-      const json = await res.json();
-      return json.resources.map((resource) => resource.resourceId);
+      try {
+        const res = await fetchSam(
+          '/api/resources/v2?format=flat&resourceTypes=enterprise-feature&roles=user',
+          _.mergeAll([authOpts(), { signal }])
+        );
+        const json = await res.json();
+        return json.resources.map((resource) => resource.resourceId);
+      } catch (error: unknown) {
```

Comment:
We might at least want to log this? Completely silencing/ignoring the error seems smelly to me

Reply:
Returning `[]` is the right approach here. If the user hasn't accepted the Terms of Service, or is unauthorized for any other reason, then they don't have access to any enterprise features!